Plot/Components/Pages/Button.test.razor

```diff
@@ -1,5 +1,28 @@
 @page "/test/button"
 
-<PageTitle>Button</PageTitle>
+<h1>Test Button</h1>
 
-<Button></Button>
\ No newline at end of file
+
+<Button Class="Test Button" Id="123" Variant="primary" OnClickFunction="TestFunction"
+Icon="fa-solid fa-ellipsis" IsDisabled=false TextAlignment="center">
+Test Text 1
+</Button>
+
+<Button Class="Test Button" Id="123" Variant="success" OnClickFunction="TestFunction"
+Icon="fa-solid fa-download" IsDisabled=false TextAlignment="right">
+Test Text 2
+</Button>
+
+<Button Class="Test Button" Id="123" Variant="danger" OnClickFunction="TestFunction"
+Icon="fa-solid fa-copy" IsDisabled=true TextAlignment="left">
+Test Text 3
+</Button>
```

Comment:
@pmdurfee Please give each of these unique IDs. An ID should always be unique per element on a page. A class would be more for a group of elements to share.